README.md

```diff
@@ -33,6 +33,9 @@ asn1 = { version = "0.18", default-features = false }
   rule, it is specific to X.509.
   ([#494](https://github.com/alex/rust-asn1/pull/494))
 
+- `GeneralizedTime` accepts fractional seconds
```

Comment:
FLAG: I think an CHANGELOG entry is needed but I don't know if this should be added there or in another section?

Reply:
I think here makes sense, since it's notionally a breaking change (as the shape of the DER changes).

Reply:
Yeah, but I'd phrase it as that GeneralizedTime has been replaced with a new type because of the abvoe.